xml/System.IO.Compression/BrotliDecoder.xml

```diff
@@ -71,19 +71,11 @@
         <param name="bytesWritten">The total number of bytes that were written in the <paramref name="destination" />.</param>
         <summary>Decompresses data that was compressed using the Brotli algorithm.</summary>
         <returns>One of the enumeration values that indicates the status of the decompression operation.</returns>
-        <remarks>
-          <format type="text/markdown"><![CDATA[
-
-## Remarks
-
-The return value can be as follows:
-- <xref:System.Buffers.OperationStatus.Done?displayProperty=nameWithType>: `source` was successfully and completely decompressed into `destination`.
-- <xref:System.Buffers.OperationStatus.DestinationTooSmall?displayProperty=nameWithType>: There is not enough space in `destination` to decompress `source`.
-- <xref:System.Buffers.OperationStatus.NeedMoreData?displayProperty=nameWithType>: The decompression action is partially done at least one more byte is required to complete the decompression task. This method should be called again with more input to decompress.
-- <xref:System.Buffers.OperationStatus.InvalidData?displayProperty=nameWithType>: The data in `source` is invalid and could not be decompressed.
-
-           ]]></format>
-        </remarks>
+        <remarks>The return value can be as follows:
```

Comment:
@carlossanlop Here's another problem with using a custom tool vs. mdoc to import the comments. Since the XML file exists for this namespace, if we use docs-api-sync to import comments and the format is different to what mdoc would import (which it is here), then it's just going to get overwritten like this the next time we run the CI job.